lib/proxy/validations.rb

```diff
@@ -39,4 +70,11 @@ def validate_record record
     record
   end
 
+  private
```

Comment:
No need to use 'private' anywhere other than in libraries. All it does is here is making it harder to test things.